opt/scopeserver/gen_test_loom.py

```diff
@@ -0,0 +1,256 @@
+import loompy as lp
+import numpy as np
+import pandas as pd
+from numpy.random import Generator, PCG64
+
+import zlib
+import json
+import base64
+import datetime
+from pathlib import Path
+
+
+def dfToNamedMatrix(df):
```

Comment:
Would be good to follow Python convention:
```suggestion
def df_to_named_matrix(df):
```
The function calls therefore should also be changed